.github/workflows/mi210_rocm621.yml

```diff
@@ -0,0 +1,170 @@
+name: Reusable MI210 workflow
+
+permissions:
+  contents: none
+
+on:
+  workflow_call:
+    inputs:
+      kokkos_version:
+        description: 'The Kokkos Core version to build'
+        default: ''
+        required: true
+        type: string
+
+jobs:
+  PR_VEGA90A_ROCM621_HIP_SERIAL_LEFT:
+    name: PR_VEGA90A_ROCM621_HIP_SERIAL_LEFT
+    runs-on: [rocm-6.2.1_openblas-0.3.28_zen3_gfx90a-latest-latest]
+    
+    steps:
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ inputs.kokkos_version }}
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          echo "GITHUB_WORKSPACE = $GITHUB_WORKSPACE"
+          mkdir "$GITHUB_WORKSPACE/kokkos/build"
+          mkdir "$GITHUB_WORKSPACE/kokkos/install"
+          cd "$GITHUB_WORKSPACE/kokkos/build"
+          HIPCC=$(which hipcc)
+          cmake -S "$GITHUB_WORKSPACE/kokkos" \
+            -B "$GITHUB_WORKSPACE/kokkos/build" \
+            -D CMAKE_CXX_COMPILER=$HIPCC \
```

Comment:
Use `hipcc` instead of `$HIPCC` as they are actually the same minus some indirection.